frontend/constants/menuLinks.ts

```diff
@@ -41,215 +41,4 @@ interface NavLink {
-    label: 'Advocates',
-    url: '/advocates#become-an-advocate'
-  }
-}
-
-const OVERVIEW_LINK: NavLink = {
-  label: 'Overview',
-  url: 'https://qiskit.org/overview'
-}
-
-const LEARN_LINK: NavLink = {
-  label: 'Learn',
-  url: 'https://qiskit.org/learn'
-}
-
-const DOCUMENTATION_LINK: NavLink = {
-  label: 'Documentation',
-  url: 'https://qiskit.org/documentation/'
-}
-
-const QISKIT_VIDEOS: NavLink = {
-  label: 'Videos',
-  url: 'https://www.youtube.com/qiskit'
-}
-
-const TUTORIALS_LINK: NavLink = {
-  label: 'Tutorials',
-  url: 'https://github.com/Qiskit/qiskit-tutorials',
-  segment: {
-    action: 'Tutorials'
-  }
-}
-
-// TODO: enable when #663 is done
-// const QISKIT_CAREERS: NavLink = {
-//   label: 'Careers',
-//   url: '#'
-// }
-
-type FooterSection = {
-  title: string,
-  elements: Array<NavLink>
-}
-
-type FooterStructure = Array<FooterSection>
-
-// Each entry represents a column
-const FOOTER_ELEMENTS: FooterStructure = [
-  {
-    title: 'Learn',
-    elements: [
-      QISKIT_EDUCATION.textbook,
-      TUTORIALS_LINK,
-      QISKIT_VIDEOS
-    ]
-  },
-  {
-    title: 'Community',
-    elements: [
-      COMMUNITY_SUB_LINKS.events,
-      QISKIT_ADVOCATES.advocates
-      // TODO: enable when #663 is done
-      // QISKIT_CAREERS
-    ]
-  },
-  {
-    title: 'Support',
-    elements: [
-      SOCIAL_MEDIA.github,
-      SOCIAL_MEDIA.stack,
-      DOCUMENTATION_LINK
-    ]
-  }
-]
-
-const STAY_CONNECTED_LINKS: FooterSection = {
-  title: 'Stay Connected',
-  elements: [
-    {
-      label: 'Twitter',
-      icon: 'LogoTwitter20',
-      url: SOCIAL_MEDIA.twitter.url
-    },
-    {
-      label: 'Slack',
-      icon: 'LogoSlack20',
-      url: SOCIAL_MEDIA.slack.url
-    },
-    {
-      label: 'Youtube',
-      icon: 'LogoYouTube20',
-      url: SOCIAL_MEDIA.youtube.url
-    },
-    {
-      label: 'Medium',
-      icon: 'LogoMedium20',
-      url: SOCIAL_MEDIA.medium.url
-    }
-  ]
-}
-
-const SECONDARY_FOOTER_LINKS: Array<NavLink> = [
-  {
-    label: 'Terms of use',
-    url: 'https://www.ibm.com/legal?lnk=flg-tous-usen'
-  },
-  {
-    label: 'Accessibility',
-    url: 'https://www.ibm.com/accessibility/us/en/?lnk=flg-acce-usen'
-  },
-  {
-    label: 'Contact',
-    url: 'mailto:hello@qiskit.camp'
-  },
-  {
-    label: 'Privacy',
-    url: 'https://quantum-computing.ibm.com/terms/privacy'
-  }
-]
-
-const HOME_LINK: NavLink = {
-  label: 'Qiskit',
-  url: 'https://qiskit.org/'
-}
-
-const GITHUB_REPOSITORY: NavLink = {
-  label: 'GitHub',
-  url: 'https://github.com/Qiskit/qiskit',
-  segment: { action: 'GitHub repository' }
-}
-
-const COMMUNITY_LINK: NavLink = {
-  label: 'Community',
-  url: 'https://qiskit.org/advocates'
-}
-
-export {
-  ORDERED_COMMUNITY_SUB_LINKS,
-  FOOTER_ELEMENTS,
-  SECONDARY_FOOTER_LINKS,
-  HOME_LINK,
-  GITHUB_REPOSITORY,
-  COMMUNITY_LINK,
-  TUTORIALS_LINK,
-  DOCUMENTATION_LINK,
-  STAY_CONNECTED_LINKS,
-  LEARN_LINK,
-  OVERVIEW_LINK,
-  NavLink
-}
+export type { NavLink }
```

Comment:
Added export `type` to resolve this lint error 👇 
<img width="963" alt="Screen Shot 2022-09-19 at 9 29 29 AM" src="https://user-images.githubusercontent.com/8097306/191067527-7458b1f1-13c9-4eee-abf5-429fbc2720ab.png">